### PR TITLE
nl: docs index page translated to Dutch

### PR DESCRIPTION
**Pull request by hursittarcan**
### Translated: 
* Docs/index.md 

@nodejs/nodejs-nl

### CODE REVIEW

**Review by Trott — COMMENTED** [review]
Rubber-stamp LGTM but I'll leave this as a comment for now in the hopes that someone who actually speaks Dutch (@nodejs/nodejs-nl) reviews it. But if that doesn't happen after a few days, I can convert this to an approval to get it landed.

File: locale/nl/docs/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### Op zoek naar API-documentatie van vorige versies?
+
+{#var nodeVersionLinks}* [Alle versies](https://nodejs.org/docs/)
+
+</div>
+
+## ES6-features
+
+De [ES6-sectie](/en/docs/es6/) beschrijft de drie ES6-feature groepen en de standaard features binnen Node.js. Het toont u ook welke versie van V8 bij een bepaalde Node.js-versie hoort.
```
Comment (AdriVanHoudt) [inline locale/nl/docs/index.md]: nit: I don't think the `u` is needed in the last sentence

**Review by AdriVanHoudt — APPROVED** [review]
LGTM, just 1 nit

File: locale/nl/docs/index.md (hunk 1 above)
Comment (timdp) [reply locale/nl/docs/index.md]: Additionally, "het" cannot refer to the antecedent "de ES6-sectie". 'Sectie' is feminine so it should be 'ze' or 'zij'.

File: locale/nl/docs/index.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+* API-referentiedocumentatie
+* ES6-features
+* Handleiding
+
+## API-referentiedocumentatie
+
+De [API-referentiedocumentatie](https://nodejs.org/api/) biedt gedetailleerde informatie over functies en objecten in Node.js. Het geeft aan welke argumenten geaccepteerd kunnen worden door een methode, de return-waarde van een methode en informatie over errors die gekoppeld zijn met deze methode. Het geeft ook aan welke methoden beschikbaar zijn binnen de verschillende versies van Node.js.
+
+Deze documentatie behandelt enkel de ingebouwde modules van Node.js. Het behandelt geen community-modules.
```
Comment (timdp) [inline locale/nl/docs/index.md]: Same issue around 'documentatie' <> 'het'/'zij' here. 😉

**Review by timdp — APPROVED** [review]

**Review by Trott — APPROVED** [review]

**Pull request merged**